gazelle/js/typescript/tsconfig.go

```diff
@@ -140,21 +161,31 @@ func parseTsConfigJSON(cm *TsConfigMap, root, configDir string, tsconfigContent
 	return &config, nil
 }
 
+// Returns the path from the bazel-root to the active tsconfig.json file
```

Comment:
What do you mean by "bazel-root"? Isn't it the directory of the tsconfig and not even related to the fact that we're working within a bazel repo?

Reply:
I slightly altered the comment to be more general. Otherwise one could just omit it